airbyte-integrations/connectors/destination-oracle/src/main/java/io/airbyte/integrations/destination/oracle/OracleNameTransformer.java

```diff
@@ -41,7 +41,7 @@ public String convertStreamName(String input) {
     }
     // prior to Oracle version 12.2, identifiers are not allowed to exceed 30 characters in length.
```

Comment:
This comment explains why identifiers are truncated to 30 characters. If we make this change, all existing users running an Oracle server prior to 12.2 cannot upgrade their connectors in the future. @sherifnada, do we have a convention about this type of thing?

Reply:
Liren, I already change to 128. This is required to run normalization.

Reply:
@marcosmarxm, ok, I will approve the PR. But will this prevent existing users to upgrade?

Reply:
is this going to require existing users to do anything? Did we already support 12.2+ and just didn't advertise it? if so this change is safe.

Reply:
> is this going to require existing users to do anything? Did we already support 12.2+ and just didn't advertise it? if so this change is safe.
I unfortunately have no information about the version of Oracle, which was supported before. We have several options for this task :
a) just add to the documentation that we only support version 12.2+ 
b) additional implementation is needed so that the user himself enters the version of the database on UI and further processing starts with this data, but in this case it is does not solve the problem with normalization (currently it works only with 128 characters for any Oracle version). 
What do you think about it?